src/main/java/io/github/railroad/platform/fabric/FabricMod.java

```diff
@@ -0,0 +1,376 @@
+package io.github.railroad.platform.fabric;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.Properties;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+/*
+ * @author lifebearrrr
+ * 
+ * Used to generate new Fabric Mod with all assets
+ * 
+ *TODO: Replace my Fabric version getter (that doesn't get the right fabric api version)
+ *with Saksham4106's Fabric version getter when it's done
+ */
+@SuppressWarnings("static-access")

```

Comment:
If you have this, then you are doing something wrong, so please fix the instances of the static access warnings.

Reply:
Sorry I forgot about this, I'll go fix it. Don't know why I added that suppress warnings annotation.